pkg/network/go/bininspect/symbols_test.go

```diff
@@ -0,0 +1,85 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024-present Datadog, Inc.
+
+//go:build linux
+
+package bininspect
+
+import (
+	"debug/elf"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/DataDog/datadog-agent/pkg/network/protocols/http/testutil"
+	"github.com/DataDog/datadog-agent/pkg/util/common"
+)
+
+func openTestElf(t *testing.T) *elf.File {
+	curDir, err := testutil.CurDir()
+	require.NoError(t, err)
+
+	// Doesn't matter which architecture we are on.
+	lib := filepath.Join(curDir, "..", "..", "usm", "testdata",
+		"site-packages", "ddtrace", "libssl.so.arm64")
```

Comment:
should it be explicitly `.arm64`?

Reply:
It doesn't matter which architecture we are on, as the comment above says.

Reply:
Expanded comment.